Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -1089,6 +1089,73 @@ void  GeomUtils::applyPositionToVertex(
 	}
 }
 
+/**********************************************************************************************/
+/*! \brief Applies a position to the point.
+*
+* \param[in] listOfPoint				A list of points to be converted.
+* \param[in] positionMatrix			A position matrix, which should be applied to the points.
+* \param[in] ammountOfPoints			Number of points.
+*
+* \return[out]							List of points with applied position.
+*/
+std::vector<carve::geom::vector<2>> GeomUtils::removeEmptyCoordinate(const std::vector<carve::geom::vector<3>>& listOfVectors3D) 
+{
+	std::vector<carve::geom::vector<2>> listOfVectors2D;
+	bool notEmptyCoord = false;
+	//Check if x Coordniates are empty.
+	for (auto vector : listOfVectors3D)
+	{
+		if (! vector.x == 0.)
+		//if (!this->GeomSettings()->areEqual(vector.x, 0.))

```

Comment:
I could not find an option how to call function areEqual from GeometrySettings.h. Can you please give me a hint?